source/js/header.js

```diff
@@ -69,6 +69,20 @@
   
   var setCountryIcon = function(cookies) {
     var countryCode = 'us';
+    
+    var localeParam = getQueryVariable("locale")
+    
+    if(localeParam) {
+      
+      splitLocale = localeParam.split('-')
+      if( splitLocale[1] ) {
+        countryCode = splitLocale[1];
+      }
+      
+      document.getElementById('country-badge').className = "flag-icon flag-icon-squared flag-icon-" + countryCode;
+      return;
+    }
+    
     if(cookies.indexOf('locale=') > -1){
```

Comment:
nit: should this be an else if? Probably doesn't matter since you have the return but just for readability since this is supposed to be a fallback.